OpenXmlPowerTools/ListItemTextGetter_de_DE.cs

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace OpenXmlPowerTools
+{
+    public class ListItemTextGetter_de_DE
+    {
+        private static string[] OneThroughNineteen = {
+            "eins", "zwei", "drei", "vier", "fünf", "sechs", "sieben", "acht",
+            "nuen", "zehn", "elf", "zwölf", "dreizehn", "vierzehn",
+            "fünfzehn", "sechzehn", "siebzehn", "achtzehn", "nuenzehn"
+        };
+
+        private static string[] Tens = {
+            "zehn", "zwanzig", "dreißig", "vierzig", "fünfzig", "sechzig", "siebzig",
+            "achtzig", "nuenzig"
+        };
+        
+        private static string[] OrdinalOneThroughNineteen = {
+            "erste", "zweite", "dritte", "vierte", "fünfte", "sechste",
+            "siebte", "achte", "nuente", "zehnte", "elfte", "zwölfte",
+            "dreizehnte", "vierzehnte", "fünfzehnte", "sechzehnte",
+            "siebzehnte", "achtzehnte", "nuenzehnte"
+        };
+
+        private static string[] OrdinalTens = {
+            "zehnte", "zwanzigste", "dreißigste", "vierzigste", "fünfzigste",
+            "sechzigste", "siebzigste", "achtzigste", "nuenzigste"
+        };
+
+        public static string GetListItemText(string languageCultureName, int levelNumber, string numFmt)
+        {
+			if (levelNumber > 19999)
+				throw new ArgumentOutOfRangeException("levelNumber", "Convering a levelNumber to ordinal text that is greater then 19 999 is not supported");
+			if (levelNumber == 0)
+				return "Zero";
+			if (levelNumber < 0)
+				throw new ArgumentOutOfRangeException("levelNumber", "Converting a negative levelNumber to ordinal text is not supported");
+
+            if (numFmt == "ordinal")
+                return GetOrdinal(levelNumber); 
+            if (numFmt == "cardinalText")
+                return GetCardinalText(levelNumber); 
+            if (numFmt == "ordinalText")
+                return GetOrdinalText(levelNumber); 
+            return null;
+        }
+
+        private static string GetOrdinal(int levelNumber)
+        {
+            string suffix;
+            if (levelNumber % 100 == 11)
+                suffix = "-te";
+            else if (levelNumber % 10 == 1)
+                suffix = "-ste";
+            else
+                suffix = "-te";
+            return levelNumber.ToString() + suffix;
+        }
+
+        private static string GetCardinalText(int levelNumber)
+        {
+            string result = "";
+
+            // Get thousands 
+            int t1 = levelNumber / 1000;
+            int t2 = levelNumber % 1000;
+            if (t1 >= 1)
+                result += (t1 == 1 ? "ein" : OneThroughNineteen[t1 - 1]) + " thausend";
```

Comment:
Please replace `thausend` with `tausend` everywhere.

Reply:
Ok, I fixed that.